CMS-Enterprise/mint-app
# [NOREF] - Fix 2 pager modal link

# [NOREF]

## Description


- Help center 2 pager article links properly to solution modal
- Updated `six` in routes to match numeral `2` of help routes


## How to test this change

1.  Verify the `Learning and Diffusion Group` help center links open properly in the 2 pager article

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
